src/node_dependencies/index.js

```diff
@@ -35,20 +38,26 @@ const isNotJunk = function (file) {
 }
 
 // Retrieve all the files recursively required by a Node.js file
-const getDependencies = async function (mainFile, srcDir, pluginsModulesPath) {
+const getDependencies = async function (mainFile, srcDir, pluginsModulesPath, moduleAllowList) {
   const packageJson = await getPackageJson(srcDir)
-
-  const state = { localFiles: new Set(), modulePaths: new Set() }
+  const state = getNewCache()
 
   try {
-    return await getFileDependencies({ path: mainFile, packageJson, state, pluginsModulesPath })
+    return await getFileDependencies({ path: mainFile, moduleAllowList, packageJson, state, pluginsModulesPath })
```

Comment:
Can we remove `moduleAllowList`? I couldn't find where it is used

Reply:
Oh, of course! It's no longer necessary. Thanks! 🙌